core/bentley/src/ProcessDetector.ts

```diff
@@ -60,13 +60,13 @@ export class ProcessDetector {
    * @note this indicates that this is a browser process started by an iTwin mobile application.
    * It will return `false` when running user-launched web browsers on a mobile device.
    */
-  public static get isMobileAppFrontend() { return this.isBrowserProcess && window.location.origin === "imodeljs://app"; }
+  public static get isMobileAppFrontend() { return this.isAndroidAppFrontend || this.isIOSAppFrontend; }

```

Comment:
@khanaffan why did you make these changes? I don't think they're right.

Reply:
As I explained this check is wrong. For debugging the source location could be http://<ip> or the custom url can change per app as two app should not have same custom url as its use for OIDC login as redirect url. We have determine the best way to ever tell about platform is just to pass it to frontend as argument. iOS App passes #port=<port> and platform=ios as argument to fronted. This is most reliable way to tell we have mobile frontend.